php-cs-test/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM composer
+
```

Comment:
I will look on the image sizes, but I'd imagine a bare composer base image would be less in size.